__tests__/browser-test.js

```diff
@@ -219,6 +219,146 @@ browsers.forEach(browser => {
       }
     });
 
+    it('uses alt attribute when forcing a 404', async () => {
```

Comment:
It would be nice to extract a few of these tests to pure unit tests on the “url function”

Reply:
Solved this by adding support for multiple conditional methods and by calling them separately when needed instead of having everything inside the same method